README.md

```diff
@@ -1,7 +1,73 @@
 # SBC-ProductHub
+This Repo consists of 2 projects:
 
+1. **Metrics report** (Within application-status folder)
+
+2. **Release Notes** (Within release-notes folder)
 ## Apps
 
 ### Application Status
+An MVP Product [Metrics Report](www.metrics.bcregistry.gov.bc.ca)
+
+The app focuses on getting the metrics for all products developed by **Ministry of Citizen Services, Business Registry Branch**.
+
+The app displays the statistics of what the team has been working on during the last sprint so that the team can reflect on their team performance and keep track of the progress of their development. 
 
 ### Release Notes
+An MVP Product [Release Notes](https://www.release-notes.bcregistry.gov.bc.ca/)
+
+The app focuses on displaying the releases for all products developed by **Ministry of Citizen Services, Business Registry Branch**.
+
+The audience can get the latest update about the Government products, what have been fixed, what are new features, what have been improved, etc
```

Comment:
Audience can get latest updates about the BC Registries products, what has been fixed, new features, improvements, etc.